src/pages/office/cat-a-mod2/components/independent-driving/independent-driving.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { IndependentDriving } from '@dvsa/mes-test-schema/categories/common';
+import {
+  OutcomeBehaviourMapProvider,
+  VisibilityType,
+} from '../../../../../providers/outcome-behaviour-map/outcome-behaviour-map';
+
+@Component({
+  selector: 'independent-driving',
+  templateUrl: 'independent-driving.html',
+})
+export class IndependentDrivingComponent implements OnChanges {
```

Comment:
Why are we having many IndependentDrivingComponents when they are not category specific?

Reply:
True, they have been copied in the past categories. Should we resolve this in a story later? Doesn't feel right to do that when this is about creating placeholder pages.